swiggy-pdp-cart/src/components/Main/Content/Cart/Cart.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
Read about `React.memo` and figure out if wrapping this component with that will prevent unnecessary re renders for this component. And if yes, then in what scenario.